templates/zerver/portico-header.html

```diff
@@ -43,12 +43,8 @@
 
             {% if find_team_link_disabled %}
             {% else %}
-            <li>
-                <a href="{{ root_domain_uri }}/accounts/find/">Find accounts</a>
-            </li>
-            <li>
-                <a href="{{ root_domain_uri }}/new">New organization</a>
-            </li>
+            <a href="{{ root_domain_uri }}/accounts/find/">Find accounts</a>
+            <a href="{{ root_domain_uri }}/new">New organization</a>
```

Comment:
Can you post before/after screenshots for these?

Reply:
Broken (currently in prod):
<img width="715" alt="broken_header_help" src="https://user-images.githubusercontent.com/7251823/127056367-b0ff6631-0b44-4136-9a32-d1b5d69d4159.png">

After:
<img width="692" alt="fixed_header_help" src="https://user-images.githubusercontent.com/7251823/127056428-9dc85aae-f63a-470b-8a36-dfbf1c2f34dd.png">